File: 002/002.py
#Nhap toa do 2 diem A, B
#tinh do dai vecto AB

def do_dai_vecto(xA, yA, xB, yB):
    AB = ((xB - xA)**2 + (yB - yA)**2)**0.5
    return "{:.4f}".format(AB)

xA, yA = map(float, input("A(xA, yA)? ").split())
xB, yB = map(float, input("B(xB, yB)? ").split())

AB = do_dai_vecto(xA, yA, xB, yB)

print(f"|AB| = {AB}")

File: 002/002.rb
#Nhap toa do 2 diem A, B
#tinh do dai vecto AB

def do_dai_vecto(xa, ya, xb, yb)
  return Math.sqrt((xb - xa)**2 + (yb - ya)**2)
end

print "A(xA, yA)? "
xa, ya = gets.split.map(&:to_f)

print "B(xB, yB)? "
xb, yb = gets.split.map(&:to_f)

ab = do_dai_vecto(xa, ya, xb, yb)

puts "|AB| = #{'%.4f' %ab}"
